Library/Homebrew/download_strategy.rb

```diff
@@ -132,6 +135,12 @@ def fossilpath
   end
 end
 
+# This is a superclass for all the Version Control System downloading strategies
+# Its subclasses include SubversionDownloadStrategy, GitDownloadStrategy,
+# CVSDownloadStrategy, MercurialDownloadStrategy, BazaarDownloadStrategy
+# and FossilDownloadStrategy
+# The naming scheme used in repositories downloaded with VCS is name--cache_tag,
+# which is useful while determining how to clear the cache in cleanup.rb
```

Comment:
How about just `This is a superclass for all the downloading strategies for Version Control Systems.`. Don't want to name specific classes or behaviour as the documentation will become outdated if the code changes.